shipyard-controller/handler/projecthandler_test.go

```diff
@@ -261,6 +284,8 @@ func TestCreateProject(t *testing.T) {
 						return nil
 					},
 				},
+				EnvConfig:             config.EnvConfig{ProjectNameMaxSize: 20},
+				RepositoryProvisioner: NewRepositoryProvisioner(""),
```

Comment:
using a real repo provisioner will according to the implementation also make real http calls.
You should use a mocked implementation here.
Also you are not testing all cases.

Reply:
It does not have any difference if you use a real one or a mocked one

Reply:
the difference is, that you cannot test the good case.